driver-examples/tycho/.gitignore

```diff
@@ -0,0 +1,2 @@
+!.classpath
```

Comment:
Is there special configuration in the .project/.classpath files that wouldn't be present when doing a 'mvn eclipse:eclipse' or importing the pom.xml in m2eclipse?  If that is the case it might be nicer to include instructions for how to import/generate these files instead of having them in git (more for future proofing if eclipse adds/removes things to these files).